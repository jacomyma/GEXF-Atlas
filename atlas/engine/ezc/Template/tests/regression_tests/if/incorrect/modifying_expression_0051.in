{* file: if/incorrect/modifying_expression_0051.in *}
{var $foo = 42, $bar = 101}
0051:{if ( $foo + (  ( $bar *= 5 ) ) )}this should never appear{/if}
