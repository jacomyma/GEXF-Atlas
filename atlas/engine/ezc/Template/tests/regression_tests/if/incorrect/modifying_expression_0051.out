mock:3:35: Unexpected operand or expression. The operand or expression modifies a variable, which is not allowed.

0051:{if ( $foo + (  ( $bar *= 5 ) ) )}this should never appear{
                                  ^
